### ISSUE TITLE
Utility script to convert Perl to JSON config

### ISSUE THREAD

**Issue by hepabolu** [bartificer/xkpasswd-js#87]
The old CLI version saved the configuration as a Perl dictionary. The current version can only handle a JSON configuration.

We need a utility script that can be used to convert any Perl configuration to JSON. This JSON config can then be imported in the current version of XKPasswd.

**Comment by podfeet** [comment]
I have a question. What was the form of the Perl dictionary created by the old CLI version? I never got it to run and we don't have access anymore. Not sure how any of us could help without knowing more. Or maybe that's why you assigned it to @bbusschots.

**Comment by hepabolu** [comment]
Yes, @bbusschots will write a utility to convert the Perl format to JSON for further processing in the new version.
